Juice/Classes/zh-Hans.lproj/GeneralPreferencesViewController.strings

```diff
@@ -0,0 +1,24 @@
+
+/*
+Contributor : Zhilian DAI 
+*/
+/* Class = "NSTextFieldCell"; title = "Rescan For Scales:"; ObjectID = "0hr-m8-QTZ"; */
+"0hr-m8-QTZ.title" = "重新扫描Emoji配置区间：";
```

Comment:
Hello, sorry for lated response, it seems some network problem occur email. Dose the "Rescan For Scales" means reload the scale configuration file?